File: src/Java/src/com/jingchen/trainer/Node.java
package com.jingchen.trainer;

public class Node {
	private int mId;
	private float mRate;

	public Node(int id, float rate) {
		mId = id;
		mRate = rate;
	}

	public int getId() {
		return mId;
	}

	public float getRate() {
		return mRate;
	}
}

File: src/C++/src/Node.cpp
/*
 * Node.cpp
 *
 *  Created on: 2014Äê10ÔÂ24ÈÕ
 *      Author: chenjing
 */

#include "Node.h"

Node::Node(int id, float rate) {
	mId = id;
	mRate = rate;
}
int Node::getId() {
	return mId;
}
float Node::getRate() {
	return mRate;
}
Node::~Node() {
}

